tests/e2etest/tests/e2e_test.go

```diff
@@ -3,7 +3,7 @@ package e2etest
 import (
 	"testing"
 
-	"github.com/GoogleContainerTools/kpt-functions-catalog/tests/e2etest/internal/runner"
```

Comment:
is it possible to fix the typo that causes the difference between the name of the module here and the name of the directory?
https://github.com/GoogleContainerTools/kpt-functions-catalog/blob/master/tests/e2e-test/go.mod

Reply:
The name of the directory has been updated.

Reply:
Ah, my mistake, I didn't see that. Thank you